packages/side-nav/src/SideNavGroup.tsx

```diff
@@ -144,15 +151,17 @@ interface SideNavGroupProps {
  * @param props.header Content that will be rendered as the component's header
  *   If a string is provided, it will be rendered with default styling as a header tag.
  * @param props.children Class name that will be applied to the component's header.
- * @param props.collapsible Determines whether or not the Group can be collapsed.
+ * @param props.collapsible Determines whether or not the Group can be collapsed. @defaultValue false
+ * @param props.defaultCollapsed Determines whether or not the Group is open by default. @defaultValue true
  */
 function SideNavGroup({
   header,
   children,
   collapsible = false,
+  defaultCollapsed = true,
   ...rest
 }: SideNavGroupProps) {
-  const [open, setOpen] = React.useState(false);
+  const [open, setOpen] = React.useState(!defaultCollapsed);
```

Comment:
I lean towards naming this `initial`, rather than `default` to more clearly indicate that its value won't be used again after the initial render and will have no effect if changed.

Reply:
Yes I like that idea, good call